lighthouse-core/audits/seo/font-size.js

```diff
@@ -236,7 +238,7 @@ class FontSize extends Audit {
       title: str_(UIStrings.title),
       failureTitle: str_(UIStrings.failureTitle),
       description: str_(UIStrings.description),
-      requiredArtifacts: ['FontSize', 'URL', 'MetaElements'],
+      requiredArtifacts: ['FontSize', 'URL', 'MetaElements', 'SourceMaps', 'ScriptElements'],
```

Comment:
I can't find where `SourceMaps` and `ScriptElements` are used in this file. Should we be finding the bundle around L200?